### ISSUE TITLE
Security warning about "spec.oneapi.com" certificate not being valid for that domain name

### ISSUE THREAD

**Issue by eero-t** [oneapi-src/level-zero-spec#309]
Trying to visit: https://spec.oneapi.com/level-zero/latest/sysman/api.html

Got:
> Firefox detected a potential security threat and did not continue to spec.oneapi.com.
> ...
>  Firefox does not trust this site because it uses a certificate that is not valid for spec.oneapi.com. The certificate is only valid for the following names: intel.com, ...

(It listed lot of sites, but not spec.oneapi.com.)

**Comment by eero-t** [comment]
Ah, problem is that the spec URL has changed, certificate lists only the new  to `*.io` site, and either there's no redirect from `*.com` address, or redirect needs valid certificate for that too, for Firefox to accept redirect.

**Comment by rwmcguir** [comment]
This is correct, new URL is https://oneapi-src.github.io/level-zero-spec/level-zero/latest/sysman/api.html
We are slowly working with the other hosting to remove or fixup forwarding.

**Comment by rwmcguir** [comment]
Closing this as the latest oneAPI Spec as of v1.4 has removed Level Zero entirely from reference. While Level Zero will remain and continue to be developed, it just is not referenced from the oneAPI Spec page at this point. Please see our github site as mentioned above for docs.
